src/main/java/am/ik/yavi/core/IncludedViolationMessages.java

```diff
@@ -0,0 +1,77 @@
+package am.ik.yavi.core;
+
+public abstract class IncludedViolationMessages {
+
+	private static IncludedViolationMessages includedViolationMessages = null;
+
+	public static IncludedViolationMessages get() {
+		if (includedViolationMessages == null) {
+			includedViolationMessages = new DefaultIncludedViolationMessages();
+		}
+		return includedViolationMessages;
+	}
+
+	public static void define(IncludedViolationMessages includedViolationMessages) {
+		IncludedViolationMessages.includedViolationMessages = includedViolationMessages;
```

Comment:
"Introducing something like a global variable" is hard to accept.
This is not thread-safe.